homework24/src/ba/bitcamp/homework24/task1/InitializeDatabase/SetupDataBase.java

```diff
@@ -0,0 +1,69 @@
+package ba.bitcamp.homework24.task1.InitializeDatabase;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * DO NOT RUN IF YOU ALREADY HAVE INITIALIZED DB.
+ * 
+ * SetupDataBase class used to initialize database with table Complaint. If
+ * table Complaint already exist it will be dropped.
+ * <P>
+ * After establishing connection to DB, table is dropped if exists and new one
+ * is created with values of id primary key, string for message with max length
+ * of 255 characters, and string for date with max length of 25 characters.
+ * <p>
+ * If there are not errors in the way message
+ * "Database 'ComplaintBook' with table Complaint initialized." will be printet
+ * to standard output.
+ *
+ *
+ * 
+ * @author boris
+ *
+ */
+public class SetupDataBase {
+
+	public static void main(String[] args) {
+
+		Connection conn = null;
+
+		try {
+
+			Class.forName("org.sqlite.JDBC");
+
+		} catch (ClassNotFoundException e) {
+			System.out.println("Could not load JDBC driver");
+			System.err.println("Error message: " + e.getMessage());
+			System.exit(1);
+		}
+
+		try {
+
+			conn = DriverManager.getConnection("jdbc:sqlite:databases/complaintbook.db");
```

Comment:
Posto aplikacija generalno uvijek ima u vecini slucajeva samo jednu konekciju uvijek napravi neku util klasu za dohvacanje konekcije npr, DatabaseConnection i metodu getConnection(). Jer to ces koristiti vise puta u aplikaciji i treba biti izvuceno na jedno mjesto.